tstypes/type/scalars.d.ts

```diff
@@ -8,4 +8,4 @@ export const GraphQLID: GraphQLScalarType;
 
 export const specifiedScalarTypes: ReadonlyArray<GraphQLScalarType>;
 
-export function isSpecifiedScalarType(type: GraphQLScalarType): boolean;
+export function isSpecifiedScalarType(type: unknown): type is GraphQLScalarType;
```

Comment:
I thought `unknown` available only in TS 3.0?

Reply:
Moved it to `any`